tests/tests_integration/test_loaders/test_resource_container_loaders.py

```diff
@@ -117,7 +117,7 @@ def test_populate_count_drop_data_node_container(
             loader.drop_data(container_id)
 
     # The DMS service is fairly unstable, so we need to rerun the tests if they fail.
-    @pytest.mark.flaky(reruns=3, reruns_delay=10, only_rerun=["AssertionError"])
+    @pytest.mark.flaky(reruns=3, reruns_delay=10, only_rerun=["AssertionError", "CogniteAPIError"])
```

Comment:
Unrelated to the rest of the PR, but I noticed that there were flakiness caused by the CogniteAPI.